src/metrics/metrics.py

```diff
@@ -86,3 +88,6 @@
     ['type'],
     namespace=PROMETHEUS_PREFIX,
 )
+
+i = Info(PROMETHEUS_PREFIX + '_info', 'Info metric')
```

Comment:
Use `namespace` instead of str concatenation.
```suggestion
INFO = Info(PROMETHEUS_PREFIX + '_info', 'Info metric')
```